files/zh-cn/mdn/community/roles_teams/index.md

```diff
@@ -5,111 +5,277 @@ slug: MDN/Community/Roles_teams
+| [**贡献者**](#贡献者)     | 遵守行为准则和贡献准则                                                                                                                                             | - 受邀请参加贡献者活动 <br> - 有资格成为组织成员       |
+| [**组织成员**](#组织成员) | - 在 GitHub 账户上启用二步认证（2FA）<br> - 启用签名提交<br><br>至少一个：<br>- 解决至少两个议题<br> - 至少贡献了两个月<br>- 在一个项目领域作出活跃贡献            | 组织级别的访问权限                                     |
+| [**维护者**](#维护者)     | 至少一个：<br>- 担任特邀专家至少六个月<br>- 了解多个项目领域<br>- 为项目的整体健康采取行动<br>- 指导其他贡献者<br>- 每月至少在项目中花费 16 小时<br>- 出席社区会议 | 批准并合并拉取请求                                     |
+| [**所有者**](#所有者)     | 仅限 Mozilla 员工                                                                                                                                                  | - 管理各种仓库的不同角色权限<br>- 添加或归档仓库和项目 |
+
+## 特殊的角色
+
+有些贡献者角色的职责更为细化，并有特殊的资格条件。这些角色包括[杰出贡献者](#杰出贡献者)、[特邀专家](#特邀专家)和[社区管理员](#社区管理员)。
+
+### 杰出贡献者
+
+杰出贡献者是那些对 MDN Web 文档做出了卓越贡献的人。他们的贡献形式包括：提出拉取请求以改进项目；在各种[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)或学习论坛上帮助社区成员；或就 GitHub 问题和拉取请求提供反馈。
+
+我们每月在 [MDN 网站](/zh-CN/)上介绍一位杰出贡献者。请参见[流程](#提名杰出贡献者)，提名某人为杰出贡献者。
+
+### 特邀专家
+
+受邀专家在 MDN 上有过贡献、参与讨论和审查的记录，或在某一专业领域有公认的知识。受邀专家负责特定主题领域或 MDN 项目的一个组成部分。他们负责审查和批准其主题或项目领域的拉取请求，回答技术问题，并维护其特定项目的总体健康状况。请参阅[流程](#提名特邀专家)，提名某人为特邀专家。
+
+除了作为[组织成员](#组织成员)的职责外，特邀专家还需要：
+
+- 遵循[审查指南](https://github.com/mdn/content/blob/main/REVIEWING.md)。
+- 审查有关他们研究主题下的拉取请求。
+- 帮助其他贡献者成为审查员。
+
+当拉取请求在受邀专家的主题领域被提出时，会自动分配专家进行审查。如果一个主题领域有多个专家，则会使用[负载平衡策略](https://docs.github.com/zh/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team#about-auto-assignment)
+
+**需求：**
+
+如果想要成为一名特邀专家，你必须至少满足以下需求之一：
+
+- 展示了对特定专题领域的深入了解。
+- 承诺对指定的主题领域负责。
+- 支持新的和偶尔加入的贡献者，并帮助拉取请求做好合并准备。
+- 参加每两个月一次的社区会议。
+
+**权利：**
+
+受邀专家会被添加到[受邀专家团队](https://github.com/orgs/mdn/teams/invited-experts-and-co-maintainers)以及相应的专题或项目团队中。受邀专家可以：
+
+- 访问所需的仓库，进行提交、拉取请求审批和合并。
+- 推荐并投票选举其他成员成为特邀专家。
+- 参加每周的 MDN Web 文档编辑电话会议。
+
+### 社区管理员
+
+社区管理者在很多方面都扮演着不同的角色，许多职责与[维护者](#维护者)相同。此外，社区经理还有以下职责：
+
+- 处理有关违反 [Mozilla 行为准则](https://github.com/mdn/mdn-community/blob/main/CODE_OF_CONDUCT.md)的报告，并决定采取适当行动。
+- 组织和开展社区活动。
+- 组织与社区相关的项目会议。
+- 确定推广 MDN 项目的媒体策略。
+- 确定并实施贡献者入职体验。
+- 对新的贡献者和用户进行入职培训。
+- 确保 MDN 项目和所有参与者的健康和福利。
+- 确定并协助实施自动化，以提高项目的可持续性。
+- 会见并确保与贡献者和合作伙伴保持健康的关系。
+- 在适当的情况下，协助问题分流和拉动请求审查。
+- 监控所有[沟通渠道](/zh-CN/docs/MDN/Community/Communication_channels)。
+- 表彰工作出色和/或为 MDN 项目做出贡献的贡献者。
+
+## 过程
+
+### 提名维护者
+
+查看作为[维护者](#维护者)的必要条件。
+
+想要提名某人为维护者，在 GitHub 上开启议题：
+
+1. 在 `mdn/mdn` 仓库的 `Issues` 选项卡中，点击右侧的 [**New issue**](https://github.com/mdn/mdn/issues/new/choose) 按钮。
+2. 在“Nominate a maintainer”（提名维护者）下，点击 **Get Started** 按钮。
+3. 在表单中填写你要提名的人员的贡献细节并提交表单。
+
+### 提名杰出贡献者
+
+查看作为[杰出贡献者](#杰出贡献者)的条件。
+
+想要提名某人为杰出贡献者，在 GitHub 上开启议题：
```

Comment:
Cloud you also split those lines for the en-US version?

https://github.com/mdn/content/blob/27e8dec205645aa69d3171e7004191a6066debf3/files/en-us/mdn/community/roles_teams/index.md?plain=1#L242-L243

Reply:
PR raised